### PR TITLE
more v243 news preparation

### PR DESCRIPTION
**Pull request by poettering**

### CODE REVIEW

File: NEWS [hunk 1]
```diff
@@ -204,7 +205,15 @@ CHANGES WITH 243 in spe:
...
         * systemd-networkd's DHCPv4 and DHCPv6 stacks can now be configured
-          seperately in the [DHCPv4] and [DHCPv6] sections.
+          separately in the [DHCPv4] and [DHCPv6] sections.
+
+        * systemd-networkd's DHCP support will now create an implicit host
+          route to the DNS server specified in the DHCP lease, in addition to
+          the routes listed explicitly in the lease. This should ensure that in
+          multi-homed systems DNS traffic leaves the systems on the interface
+          that acquired the DNS server information even if other routes such as
+          default routes exist. This behaviour may be turned off with the new
```
Comment (yuwata) [inline NEWS]: `can be turned on`, as now it defaults to off, unfortunately.

File: NEWS [hunk 2]
```diff
@@ -249,6 +258,10 @@ CHANGES WITH 243 in spe:
           been renamed to LinkLayerAddress=, and it now allows configuration of
           IP addresses, too.
 
+        * A new tool systemd-network-generator has been added that may generate
+          .network files from IP configuration specified on the kernel command
```
Comment (yuwata) [inline NEWS]: also generates .netdev and .link files.

File: NEWS (hunk 1 above)
Comment (poettering) [reply NEWS]: oh why? let's turn it on by default, it's so useful!

**Comment by poettering** [comment]
force pushed, made the changes @yuwata suggested.

(btw @yuwata i might have missed some items in the NEWS file, would be cool if you could have a look and maybe push a follow-up PR if you find something missing?)

File: NEWS (hunk 1 above)
Comment (yuwata) [reply NEWS]: It breaks Ubuntu CIs unfortunately.

**Comment by yuwata** [comment]
> force pushed, made the changes @yuwata suggested.

LGTM.

> (btw @yuwata i might have missed some items in the NEWS file, would be cool if you could have a look and maybe push a follow-up PR if you find something missing?)

OK. But currently I do not have any entry missed.

**Pull request merged**